gc/verbose/VerboseWriterFileLoggingSynchronous.cpp

```diff
@@ -117,7 +120,16 @@ MM_VerboseWriterFileLoggingSynchronous::openFile(MM_EnvironmentBase *env)
 	extensions->getForge()->free(filenameToOpen);
 	
 	omrfile_printf(_logFileDescriptor, getHeader(env), version);
-	
+
+	if (printInitializedHeader){
```

Comment:
Formatting for this block, see comment for `VerboseWriterFileLoggingBuffered.cpp`